SayTheirNames/Source/Controller/Home/HomeController.swift

```diff
@@ -74,28 +73,16 @@ class HomeController: UIViewController, ServiceReferring {
                          .init(name: "MISSOURI"),
                          .init(name: "TEXAS"),
                          .init(name: "NEW YORK")]
-        
-        var people: [Person] = []
-        let df = DateFormatter()
-        df.dateFormat = "dd.MM.yyyy"
-        for i in 0..<10 {
-            people.append(Person(
-                id: "id\(i)",
-                fullName: "George Floyd \(i)",
-                age: i,
-                childrenCount: i,
-                date: df.date(from: "25.05.2020") ?? Date(),
-                location: "",
-                media: ["man-in-red-jacket-1681010"],
-                bio: "",
-                context: "",
-                donations: [],
-                petitions: []))
-        }
-        
         locationsDataSource.setLocations(locations)
-        peopleDataSource.setPeople(people)
-        
+
+        // FIXME: This should be setup in a better place, for now this loads out data
+        self.service.network.fetchPeople { [weak self] (p) in
+            guard let people = p?.all else { return }
+            
+            self?.peopleDataSource.setPeople(people)
+            self?.peopleCollectionView.reloadData()
```

Comment:
🎉 🎉 🎉

Reply:
🔥

Reply:
Is the fact that this is a `network` really relevant to the HomeController? Maybe a bit pedantic but it feels like it's not really what we want to convey to the controller.